src/it/resources/com/google/checkstyle/test/chapter4formatting/rule462horizontalwhitespace/InputGenericWhitespace.java

```diff
@@ -13,8 +13,30 @@ void meth()
     {
         List<Integer> x = new ArrayList<Integer>();
         List<List<Integer>> y = new ArrayList<List<Integer>>();
-        List < Integer > a = new ArrayList < Integer > (); // warn
-        List < List < Integer > > b = new ArrayList < List < Integer > > (); // warn
+        List < Integer > a = new ArrayList < Integer >();
+        // 6 violations above:
+        //                    ''\<' is followed by whitespace.'
+        //                    ''\<' is preceded with whitespace.'
+        //                    ''\>' is preceded with whitespace.'
+        //                    ''\<' is followed by whitespace.'
+        //                    ''\<' is preceded with whitespace.'
+        //                    ''\>' is preceded with whitespace.'
```

Comment:
Check my comment https://github.com/checkstyle/checkstyle/pull/15181#discussion_r1667106914

Reply:
done

https://github.com/checkstyle/checkstyle/blob/1e69273b5be66b96283ab28b3da5035b76a140a0/src/it/resources/com/google/checkstyle/test/chapter4formatting/rule462horizontalwhitespace/InputGenericWhitespace.java#L17-L23